README.md

```diff
@@ -245,16 +245,36 @@ Custom remappings are defined on a per-mode basis.
 * Bind `:` to show the command palette:
 
 ```json
-"vim.normalModeKeyBindingsNonRecursive": [
-    {
-        "before": [":"],
-        "commands": [
-            {
-                "command": "workbench.action.showCommands",
-            }
-        ]
-    }
-]
+    "vim.normalModeKeyBindingsNonRecursive": [
```

Comment:
Awesome! Would you mind updating the other examples to use the more simplified version?

I'd also be sure to leave examples of how to specify multiple commands and such with both notations.

Reply:
Do you have an example of a command which also take arguments? I tried finding one to show a "complete" object but didn't succeed. 

In may case I'll change the README in a way that suggests the string is the default, and then when the command takes additional arguments you can use the fully fledged object.